internal/bminventory/inventory.go

```diff
@@ -5342,6 +5342,21 @@ func (b *bareMetalInventory) V2DownloadInfraEnvFiles(ctx context.Context, params
 			return common.GenerateErrorResponder(err)
 		}
 		filename = fmt.Sprintf("%s-%s", params.InfraEnvID, params.FileName)
+	case "static-network-config":
+		if infraEnv.StaticNetworkConfig != "" {
+			netFiles, err := b.staticNetworkConfig.GenerateStaticNetworkConfigData(ctx, infraEnv.StaticNetworkConfig)
+			if err != nil {
+				b.log.WithError(err).Errorf("Failed to create static network config data")
+				return common.GenerateErrorResponder(err)
+			}
+			buffer, err := staticnetworkconfig.GenerateStaticNetworkConfigArchive(netFiles)
+			if err != nil {
+				b.log.WithError(err).Errorf("Failed to create static network config archive")
+				return common.GenerateErrorResponder(err)
+			}
+			content = buffer.String()
+			filename = fmt.Sprintf("%s-%s.tar", *infraEnv.Name, params.FileName)
```

Comment:
The ipxe-script above uses the infra-env ID rather than the name in the filename.  Any reason to change that here?

Reply:
Thank you for catching! Sorry, originally changed it to the infra-env name for easy debugging. Changed it to the infra-env ID